src/components/settings-row/settings-row.scss

```diff
@@ -1,4 +1,4 @@
-@import 'colors';
+@import "./../../style-config/colors";
```

Comment:
this was done on purpose to try and support the ability for developers to override the variables setup in this directory (see https://github.com/Sage/carbon-factory/blob/master/src/gulp/build.js#L256-L257 )

I'm not sure anyone actually uses this feature though so might be worth removing it, in which case relative paths here would be fine.